### ISSUE TITLE
Error when using "enabled: false" together with "match_path"

### ISSUE THREAD

**Issue by prcr** [Guts/mkdocs-rss-plugin#104]
I get the following error if I try to disable the RSS feed generation using the new option `enabled: false`:

```bash
$ mkdocs build
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/prcr/Codacy/git/docs/site 
ERROR   -  Error reading page 'index.md': 'GitRssPlugin' object has no attribute 'match_path_pattern' 
Traceback (most recent call last):
  File "/home/prcr/.local/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
  File "/usr/lib/python3/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/prcr/.local/lib/python3.7/site-packages/mkdocs/__main__.py", line 152, in build_command
    build.build(config.load_config(**kwargs), dirty=not clean)
  File "/home/prcr/.local/lib/python3.7/site-packages/mkdocs/commands/build.py", line 271, in build
    _populate_page(file.page, config, files, dirty)
  File "/home/prcr/.local/lib/python3.7/site-packages/mkdocs/commands/build.py", line 175, in _populate_page
    'page_content', page.content, page=page, config=config, files=files
  File "/home/prcr/.local/lib/python3.7/site-packages/mkdocs/plugins.py", line 94, in run_event
    result = method(item, **kwargs)
  File "/home/prcr/.local/lib/python3.7/site-packages/mkdocs_rss_plugin/plugin.py", line 183, in on_page_content
    if not self.match_path_pattern.match(page.file.src_path):
AttributeError: 'GitRssPlugin' object has no attribute 'match_path_pattern'
```

This happens because when we have `enabled: false`, the plugin completely skips the initialization of the config object:

https://github.com/Guts/mkdocs-rss-plugin/blob/main/mkdocs_rss_plugin/plugin.py#L87-L88

However, during the event `on_page_content`, the plugin tries to read the uninitialized `self.match_path_pattern`:

https://github.com/Guts/mkdocs-rss-plugin/blob/main/mkdocs_rss_plugin/plugin.py#L182-L184

I believe that all logic in this event should also be skipped instead when `enabled: false`. :thinking:

**Comment by prcr** [comment]
If no one picks this up in the meanwhile I may be able to open a pull request to propose a fix in the next few days.

**Comment by Guts** [comment]
Thanks @prcr for your interest and taking the time to report.
This plugin definitely lacks of serious unit testing...

I'll investigate shorty.

**Comment by Guts** [comment]
Sorry for the delay, I could not get enough free time to work on this.... but fortunately, @dcode came and fixed it :partying_face:!

Released as 0.20.1 right now.
